### ISSUE TITLE
High CPU load

### ISSUE THREAD

**Issue by ExSign** [buzz/volctl#52]
Since the new AUR version update to 0.9.0-2, I see a considerable increase in CPU load. 22-23% in htop, while idling. In previous versions I would not see anything, CPU load close to 0% for volctl.

If there is any further information that would help tracking down the source of that, please just tell me and I will try to provide that.

![grafik](https://user-images.githubusercontent.com/43409436/128033714-7a77e798-bdd4-446c-abd7-a3c36f63a411.png)

**Comment by raven2cz** [comment]
I can confirm this issue too. I have load around 3-4% which seems to be hot loop cycle. I will try to look on it.

**Comment by raven2cz** [comment]
I found and fix the error with high cpu load.

I attached images of profiler.

![volctl-cpu-profiler-1](https://user-images.githubusercontent.com/20504243/129862458-624447a4-81c8-4ed1-952e-535fc1bfb9d9.jpg)

![volctl-cpu-profiler-2](https://user-images.githubusercontent.com/20504243/129862477-24db84c8-3da0-4702-8e48-4f78d9f31619.jpg)

Problem is _wrapper for pulsectl, which is called and takes the control for long time and cpu.

I continued and found the problem with calling _pulse_pool mechanism in the `pulsectl.py`

The handling of this library is `pulsemgr.py`in run() method calling `self._pulse.event_listen()`.
The blocking calling has set 0.5 which is wrongly processed by pulsectl library. There are two choices to increase this number to 500-1000, or use non-blocking mechanism with 0. 

I select non-blocking mechanism, the cpu usage is now <0.01%.

```
diff --git a/volctl/pulsemgr.py b/volctl/pulsemgr.py
index 2b23e3a..9162561 100644
--- a/volctl/pulsemgr.py
+++ b/volctl/pulsemgr.py
@@ -43,7 +43,7 @@ class PulsePoller(threading.Thread):
             with self._pulse_hold:
                 self._pulse_lock.acquire()
             try:
-                self._pulse.event_listen(0.5)
+                self._pulse.event_listen(0)
                 if self.quit:
                     break
             except PulseDisconnected:
``` 

This is a critical bug of usage of whole volctl. I hope that buzz will find little time to merge PR, or make its own correct implementation of this bug.

I add this correction to my PR which is already added to the github PRs.
